lib/ecto/repo.ex

```diff
@@ -443,12 +443,23 @@ defmodule Ecto.Repo do
 
   ## Examples
 
+      # Use a single atom to preload an association
       posts = Repo.preload posts, :comments
-      posts = Repo.preload posts, comments: :permalinks
+
+      # Use a list of atoms to preload multiple associations
       posts = Repo.preload posts, [:comments, :authors]
-      posts = Repo.preload posts, [comments: [:replies, :likes], :authors]
-      posts = Repo.preload posts, comments: from(c in Comment, order_by: c.published_at)
 
+      # Use a keyword list to preload nested associations as well
+      posts = Repo.preload posts, [comments: [:replies, :likes], authors: []]
+
+      # Use a keyword list to customize how associations are queried
+      posts = Repo.preload posts, [comments: from(c in Comment, order_by: c.published_at)]
+
+      # Use a two-element tuple for a custom query and nested association definition
+      query = from c in Comment, order_by: c.published_at
+      posts = Repo.preload posts, [comments: {query, [:replies, :likes]}]
+
+  Note: Preloading nested associations in a custom query definition does work as well.
```

Comment:
What about just saying:

> The query given to preload may also preload its own entries.

Reply:
That's sounds better. Just maybe use `its own associations.`? To me this would be more consistent with the terminology used in the code comments.